### DISCUSSION TITLE
Pico W internal light

### DISCUSSION THREAD

**Discussion by zackify** [microsoft/devicescript#403] [Q&A] [answered]
I don't see a way to turn on the internal light on the pico w with the startLightBulb command. I searched the issues a lot and the docs. Sorry if this was explained elsewhere.

**Comment by pelikhan** [comment]
You are right, looks like we haven't updated the hardware configuration with led info 

https://microsoft.github.io/devicescript/devices/rp2040/pico-w

**Comment by pelikhan** [comment, marked as answer]
It looks like the LED is connected to the wireless chip, not the RP2040, on the pico-w so we cannot control it with the startLightbulb function.

**Reply by pelikhan** [reply]
Tracking issue: https://github.com/microsoft/devicescript/issues/404

**Comment by pelikhan** [comment]
@mmoskal can we update the board description to callout that the LED is not available?

**Comment by pelikhan** [comment]
You can also try to use `setStatusLight` https://microsoft.github.io/devicescript/developer/status-light

**Reply by zackify** [reply]
That doesn’t do anything. Already tried. Would be nice to get support for it

**Reply by pelikhan** [reply]
Tracking this at https://github.com/microsoft/devicescript/issues/404 .

**Reply by pelikhan** [reply]
(the move to discussions may have been a bit silly)

**Reply by pelikhan** [reply]
it looks like it's not properly wired up yet. we already use the LED with the wifi service so this should be possible.
https://github.com/microsoft/devicescript-pico/blob/main/src/wifi_impl.c#LL162C6-L162C20
